### PR TITLE
Fixed a fatal bug on the trigger_action.py

### PR DESCRIPTION
**Pull request by dbrown99c**
Line 40 in trigger_action.py appeared to have an old import reference to logging so I updated it to match new import of LOGGER.

### CODE REVIEW

File: depthai_sdk/setup.py [hunk 1]
```diff
@@ -1,44 +1,51 @@
...
 
 setup(
     name='depthai-sdk',
     version='1.15.0',
     description='This package provides an abstraction of the DepthAI API library.',
-    long_description=io.open("README.md", encoding="utf-8").read(),
+    long_description=open('README.md').read(),
     long_description_content_type="text/markdown",
-    url='https://github.com/luxonis/depthai/tree/main/depthai_sdk',
+    url='https://github.com/dbrown99c/depthai/tree/main/depthai_sdk',
```
Comment (jakaskerl) [inline depthai_sdk/setup.py:17]: Can't approve that. Please fix this first before we can merge.

File: depthai_sdk/setup.py [hunk 2]
```diff
@@ -1,44 +1,51 @@
...
+            'av'
+        ],
+        "test": [
+            'pytest'
+        ]
     },
     project_urls={
-        "Bug Tracker": "https://github.com/luxonis/depthai/issues",
-        "Source Code": "https://github.com/luxonis/depthai/tree/main/depthai_sdk",
+        "Source Code": "https://github.com/dbrown99c/depthai",
```
Comment (jakaskerl) [inline depthai_sdk/setup.py:48]: This as well

File: depthai_sdk/src/depthai_sdk/trigger_action/trigger_action.py [hunk 3]
```diff
@@ -37,7 +37,7 @@ def new_packet_trigger(self, packet: FramePacket) -> None:
         if self.trigger.condition(packet):
             trigger_time = datetime.now()
             if trigger_time - self.last_trigger_time > self.trigger.cooldown:
-                logging.debug(f'Triggered at {trigger_time}')
+                LOGGER.debug(f'Triggered at {trigger_time}')
```
Comment (jakaskerl) [inline depthai_sdk/src/depthai_sdk/trigger_action/trigger_action.py:40]: Thanks!

**Review by jakaskerl — CHANGES_REQUESTED** [review]

File: depthai_sdk/setup.py (hunk 1 above)
Comment (dbrown99c) [reply depthai_sdk/setup.py]: removed all the extraneous commits apologies thank you

File: depthai_sdk/setup.py (hunk 2 above)
Comment (dbrown99c) [reply depthai_sdk/setup.py]: removed all the extraneous commits apologies thank you

**Review by dbrown99c — COMMENTED** [review]
Removed irrelevant commits

**Review by jakaskerl — APPROVED** [review]
LGTM! Thanks.

**Pull request merged**